frontend/src/components/Pages/productUploadForm/ProductUploadForm.tsx

```diff
@@ -49,90 +49,125 @@ export default function ProductUploadForm() {
     }
     setFormValues((prev) => ({ ...prev, [name]: newValue }));
   };
+
   const handleImageChange = (e: React.ChangeEvent<HTMLInputElement>) => {
     if (e.target.files && e.target.files[0]) {
       setFormValues((prev) => ({ ...prev, picture: e.target.files[0] }));
     }
   };
+
   const handleSubmit = (e: React.FormEvent) => {
     e.preventDefault();
     console.log(formValues);
   };
+
   return (
-    <form onSubmit={handleSubmit}>
+    <form className="upload-form" onSubmit={handleSubmit}>
+      <label htmlFor="name">Name</label>
       <input
+        id="name"
+        className="upload-form-input"
         type="text"
         name="name"
         placeholder="Your Name"
         onChange={handleInputChange}
         value={formValues.name}
       />
-      <input type="file" name="picture" onChange={handleImageChange} />
+
+      <label htmlFor="picture">Upload Product Image</label>
+      <input
+        id="picture"
+        className="upload-form-input"
+        type="file"
+        name="picture"
+        onChange={handleImageChange}
+      />
+
+      <label htmlFor="description">Description of Product</label>
       <textarea
+        id="description"
         name="description"
         placeholder="Describe your produce"
         onChange={handleInputChange}
         value={formValues.description}
+        className="upload-form-textarea"
       />
+
+      <label htmlFor="postcode">Postcode</label>
       <input
+        id="postcode"
+        className="upload-form-input"
         type="text"
         name="postcode"
         placeholder="Postcode"
         onChange={handleInputChange}
         value={formValues.postcode}
       />
+
+      <label htmlFor="price">Price in £</label>
       <input
+        id="price"
+        className="upload-form-input"
         type="number"
         name="price"
-        placeholder="Price in £"
+        placeholder="9"
```

Comment:
Should not it be "price"?

Reply:
Ahhh yes, I was just experimenting with that to see the behaviour. I've just changed it now!